flutter_inappwebview/pubspec.yaml

```diff
@@ -86,4 +90,4 @@ flutter:
 
 false_secrets:
   - /test_node_server/*.pem
-  - /test_node_server/*.pfx
\ No newline at end of file
+  - /test_node_server/*.pfx
```

Comment:
I haven't touched that file, it seems my IDE did, let me know if I should revert